### ISSUE TITLE
RuntimeError occur in RFRNet_Smaller_Hole.py

### ISSUE THREAD

**Issue by QueeneTam** [jingyuanli001/RFR-Inpainting#18]
Thanks for author's contribution for this great work! But I got some trouble when training my own model. 

RuntimeError occur in RFRNet_Smaller_Hole.py → RFRModule.forward(): When i is 1, 'h.shape' is 'torch.Size([6, 256, 68, 68])'  and 'h_dict[enc_h_key].shape' is 'torch.Size([6, 128, 67, 67])', making 'h = torch.cat([h, h_dict[enc_h_key]], dim=1)' throws error 'invalid argument 0: Sizes of tensors must match except in dimension 1. Got 67 and 68 in dimension 2'. 

Hope you can give me some ideas to fix this error. Thanks a lot!

**Comment by QueeneTam** [comment]
I found the problem caused by dimension miasmatch, so I sovle this problem by replacing
   ' self.Pconv2 = PartialConv2d(64, 64,  5, 1, 3, multi_channel=True, bias=False) '  
   ' self.Pconv22 = PartialConv2d(64, 64, 5, 1, 3 multi_channel=True, bias=False) ' 
with
   ' self.Pconv2 = PartialConv2d(64, 64, 5, 1, 2, multi_channel=True, bias=False) ' 
   ' self.Pconv22 = PartialConv2d(64, 64, 5, 1, 1, multi_channel=True, bias=False) '

**Comment by jingyuanli001** [comment]
Good catch! Thank you for this question. I'll update the code to address this issue.
